package.json

```diff
@@ -1,8 +1,8 @@
 {
-  "name": "chef-workstation",
-  "productName": "Chef Workstation",
+  "name": "chef-workstation-app",
+  "productName": "Chef Workstation App",
```

Comment:
Oh man. This is tricky. `productName` is the easiest thing to use when making the GUI talk about itself. That's the string usually used for "About X" in the menu and window title and such.

I know this package is the App component, but maybe the `productName` would remain "Chef Workstation" because the App is the GUI aspect of the Workstation product?

Stupid language. Words are hard.

Reply:
Faces were pointed at each other and we concluded that we can add `displayName` key with the value of `"Chef Workstation"` and use _that_ for display purposes. This way the package metadata is still properly scoped to the App _component_ while also providing the string for the Workstation product that should be _displayed_ in the GUI.

Reply:
The commit message has more info, but because we want the installed application name to be "Chef Workstation App", we have to keep the productName matching that. 

Instead, I added a displayName key, and reference that in places where we're showing the name - each of those is referencing the Chef Workstation product installation, and not the Chef Workstation App.

Reply:
From learnings yesterday regarding rebranding an Electron-based application, the `productName` is what will be used for the macOS pull-down menu (upper-left between the 🍎 and the "File" menus) and for VoiceOver announcing focus (by click or keyboard nav) on the menubar/tray icon. I think it is OK for both of those to also be "Chef Workstation App" but figured I would document that here.

Reply:
Thanks for digging into this @robbkidd.  I think "Chef Workstation App" for VoiceOver is   good  in this case - it gives clarity, particularly when we can say "navigate to the chef workstation app tray icon".

Reply:
pull-down menu is less good, but since it fits the accessibility case for the icon, maybe it's workable?